test/helm/rbac_drift_test.go

```
@@ -0,0 +1,218 @@
+}
+
+func helmManagerRules(t *testing.T, chartDir string) []rbacv1.PolicyRule {
+	t.Helper()
+	// Render with every feature toggle enabled so the chart's
+	// rendered RBAC reflects the union of all conditional rules.
+	// The kustomize source-of-truth role.yaml is unconditional, so a
+	// fair parity comparison must compare against the chart's
+	// maximum-RBAC rendering. New conditional toggles that gate RBAC
+	// must be added here, otherwise this test will start failing
+	// when a kubebuilder marker references the gated resource.
+	cmd := exec.Command("helm", "template", "drift-test", chartDir,
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)